docs/content/development/extensions-core/mysql.md

```diff
@@ -53,3 +53,22 @@ Make sure to [include](../../operations/including-extensions.html) `mysql-metada
   packaged in a separate tarball that can be downloaded from [here](http://druid.io/downloads.html).
   You can also get it using [pull-deps](../../operations/pull-deps.html), or you can build
   it from source code; see [Build from Source](../build.html).
+
+
+## Encrypting MySQL connections
+  This extension provides support for encrypting MySQL connections. To get more information about encrypting MySQL connections using TLS/SSL in general, please refer to this [guide](https://dev.mysql.com/doc/refman/5.7/en/using-encrypted-connections.html).
+
+## Configuration
+
+|Property|Description|Default|Required|
+|--------|-----------|-------|--------|
+|`druid.metadata.mysql.ssl.useSSL`|Enable SSL|`false`|no|
+|`druid.metadata.mysql.ssl.trustCertificateKeyStoreUrl`|The file path to the trusted root certificate key store.|none|no|
+|`druid.metadata.mysql.ssl.trustCertificateKeyStoreType`|The type of the key store where trusted root certificates are stored.|none|no|
+|`druid.metadata.mysql.ssl.trustCertificateKeyStorePassword`|Password for the trust store.|none|no|
+|`druid.metadata.mysql.ssl.clientCertificateKeyStoreUrl`|The file path URL to the client certificate key store.|none|no|
+|`druid.metadata.mysql.ssl.clientCertificateKeyStoreType`|The type of the key store where the client certificate is stored.|none|no|
+|`druid.metadata.mysql.ssl.clientCertificateKeyStorePassword`|Password for the client key store.|none|no|
```

Comment:
is it possible in some cases to have same values for “trustCertificateKeyStoreUrl” and “clientCertificateKeyStoreUrl” .. their types and passwords ? in that case we could default client* properties to be same as trust* properties

Reply:
ignore it, i think in  all cases they would be independent.